stargate/Dockerfile

```diff
@@ -23,9 +22,9 @@ FROM alpine:latest
 # TODO: Unsure if needed
 RUN apk --no-cache add ca-certificates
 COPY --from=builder \
-  /home/rust/src/target/x86_64-unknown-linux-musl/release/stargate \
+  /home/rust/src/target/x86_64-unknown-linux-musl/release/apollo-stargate \
   /usr/local/bin/
 
 # User can provide an alternate port via "-e 'PORT=<your port>'" when running the container.
 # PORT defaults to 8080 otherwise.
-CMD /usr/local/bin/stargate --address=http://0.0.0.0:${PORT:-8080} --manifest=/graph
+CMD /usr/local/bin/apollo-stargate --port=${PORT:-8080} --manifest=${CSDL:-"/graph.graphql"}
```

Comment:
There's a specific reason `PORT` is a variable here and that the internal CSDL path to the container was not, since it's nearly the de-facto standard for configuring the listening/bind port for many Docker containers (common practice).

I don't think that customizing the path where the CSDL is mounted in the container is a necessary feature; the user shouldn't need to change the expected locations of things inside this runtime image, and they can configure the volume mount with `-v /local/path/to/their/csdl.graphql:/graph` when running the image (or the appropriate instruction in their container manifest/specification).

I don't particularly care where the graph is mounted _inside_ (including whether it has the `.graphql` extension or not), but I think it should be our implementation detail.

Reply:
Makes sense! I'll fix.